include/functions.h

```diff
@@ -905,10 +905,10 @@ Actor* Actor_Spawn(ActorContext* actorCtx, GlobalContext* globalCtx, s16 index,
                    s16 rotZ, s16 sParm10);
 ActorInit* Actor_LoadOverlay(ActorContext* actorCtx, s16 index);
 Actor* Actor_SpawnAsChildAndCutscene(ActorContext* actorCtx, GlobalContext* globalCtx, s16 index, f32 x, f32 y, f32 z,
-                                     s16 rotX, s16 rotY, s16 rotZ, s16 variable, u32 cutscene, s32 param_12,
+                                     s16 rotX, s16 rotY, s16 rotZ, s32 variable, u32 cutscene, s32 param_12,
                                      Actor* parent);
 Actor* Actor_SpawnAsChild(ActorContext* actorCtx, Actor* parent, GlobalContext* globalCtx, s16 index, f32 x, f32 y,
-                          f32 z, s16 rotX, s16 rotY, s16 rotZ, s16 variable);
+                          f32 z, s16 rotX, s16 rotY, s16 rotZ, s32 variable);
```

Comment:
Huh? Variable for these is double the regular length?

That doesn't sound right, did you get a fake match earlier?

Reply:
I'm working in decomping those, and I can confirm the `s32` is needed here.